community/contributing/docs/github-changes.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "Make a Change Using the GitHub UI"
```

Comment:
GitHub Desktop client/UI? This is not the web UI that's being discussed in this and might be confusing for people.